notifier.rst

```diff
@@ -0,0 +1,562 @@
+            'notifier' => [
+                'texter_transports' => [
+                    'twilio' => '%env(TWILIO_DSN)%',
+                ],
+            ],
+        ]);
+
+.. _notifier-chatter-dsn:
+
+Chat Channel
+~~~~~~~~~~~~
+
+The chat channel is used to send chat messages to users by using
+:class:`Symfony\\Component\\Notifier\\Chatter` classes. Symfony provides
+integration with these chat services:
+
+========  =============================  ============================================
+Service   Package                        DSN
+========  =============================  ============================================
+Slack     ``symfony/slack-notifier``     ``slack://TOKEN@default?channel=CHANNEL``
+Telegram  ``symfony/telegram-notifier``  ``telegram://TOKEN@default?channel=CHAT_ID``
+========  =============================  ============================================
+
+Chatters are configured using the ``chatter_transports`` setting:
+
+.. code-block:: bash
+
+    # .env
+    SLACK_DSN=slack://TOKEN@default?channel=CHANNEL
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/notifier.yaml
+        framework:
+            notifier:
+                chatter_transports:
+                    slack: '%env(SLACK_DSN)%'
+
+    .. code-block:: xml
+
+        <!-- config/packages/notifier.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony
+                https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+            <framework:config>
+                <framework:notifier>
+                    <framework:chatter-transport name="slack">
+                        %env(SLACK_DSN)%
+                    </framework:chatter-transport>
+                </framework:notifier>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        # config/packages/notifier.php
+        $container->loadFromExtension('framework', [
+            'notifier' => [
+                'chatter_transports' => [
+                    'slack' => '%env(SLACK_DSN)%',
+                ],
+            ],
+        ]);
+
+Email Channel
+~~~~~~~~~~~~~
+
+The email channel uses the :doc:`Symfony Mailer </mailer>` to send
+notifications using the special
+:class:`Symfony\\Bridge\\TwigBridge\\Mime\\NotificationEmail`. It is
+required to install the Twig bridge along with the Inky and CSS Inliner
+Twig extensions:
+
+.. code-block:: terminal
+
+    $ composer require symfony/twig-pack twig/cssinliner-extra twig/inky-extra
+
+After this, configure the mailer. You can also set the default "from"
```

Comment:
maybe add an anchor to mailer documentation
is it is a "relative" new component, it can not be installed on some project